package.nls.json

```diff
@@ -9,6 +9,8 @@
     "taipy.config.commands.taipy.config.file.create": "Taipy: Create New Config file",
     "taipy.config.commands.taipy.details.showLink": "Taipy: Show Source",
     "taipy.config.commands.taipy.explorer.file.setMainModule": "Taipy: Set as main module",
+    "taipy.config.commands.taipy.perspective.duplicateNode": "Taipy: Duplicate entity",
```

Comment:
I'm...not sure you're talking Entities here. Configuration elements, right?

Reply:
indeed
"Duplicate Configuration element" ?

Reply:
or "Duplicate element" @FabienLelaquais ?

Reply:
"Taipy: Duplicate" is enough